src/seam-connect/client.ts

```diff
@@ -38,17 +38,26 @@ export interface SeamClientOptions {
 export const getSeamClientOptionsWithDefaults = (
   apiKeyOrOptions?: string | SeamClientOptions
 ): SeamClientOptions => {
-  const seamClientDefaults = {
-    apiKey: globalThis?.process?.env?.SEAM_API_KEY ?? undefined,
+  const providedOptions =
+    typeof apiKeyOrOptions === "string"
+      ? { apiKey: apiKeyOrOptions }
+      : apiKeyOrOptions ?? {}
+
+  return {
+    apiKey:
```

Comment:
There needs to be a way to set a null api key to prevent the use of `SEAM_API_KEY` env var. I think passing `null` should be allowed, but undefined or missing is the same. This is how default args works, so it's consistent with how JS args are expected to work: `({ foo = 123 })` only assigns `123` if missing or `undefined`, but will allow `foo: null`.